src/metabase/driver/query_processor.clj

```diff
@@ -92,6 +92,12 @@
                                                   msg)))
                                             explanation))))
 
+
+;; +-------------------------------------------------------------------------------------------------------------------+
```

Comment:
is there any actual way that the 3 semicolons impacts something?  i'm curious what actual effect there is in changing this.